src/AlpineSkiHouse.Web/Events/PassPurchase.cs

```diff
@@ -0,0 +1,10 @@
+﻿using MediatR;
+
+namespace AlpineSkiHouse.Events
+{
+    public class PassPurchase
+    {
+        public int PassTypeId { get; set; }
+        public int CardId { get; set; }
```

Comment:
I think if we were in the real world of things we might want to put the complete details in, either here or in the Purchase completed events, to capture things like the price paid for the pass, discount code used, etc. Do you think we could either a) ignore it completely or b) make a note in the source comments or in the book? Your chapter is already stoopid-hyooge, so I don't want to push too much in that would cloud the book focus, but at the same time we might want to note it (could even be a two-sentence side bar).